src/main/java/com/snowplowanalytics/snowplow/tracker/emitter/EventStore.java

```diff
@@ -6,11 +6,13 @@
 
 public interface EventStore {
 
-    boolean add(TrackerPayload trackerPayload);
+    boolean addEvent(TrackerPayload trackerPayload);
 
-    List<TrackerPayload> removeEvents(int numberToRemove);
-
-    int getSize();
+    BatchPayload getEventBatch(int numberToRemove);
 
     List<TrackerPayload> getAllEvents();
+
+    void cleanupAfterSendingAttempt(boolean successfullySent, long batchId);
+
+    int getSize();
```

Comment:
maybe this could be just `size()` like in the Java collections.

Reply:
I wondered that. Normal style is to have `get` in the name if it returns something? But in this case it's doing the same as `size()` and so would be clearer just called that?